src/group_theory/perm/basic.lean

```diff
@@ -166,6 +167,9 @@ def subtype_perm (f : perm α) {p : α → Prop} (h : ∀ x, p x ↔ p (f x)) :
   λ _, by simp only [perm.inv_apply_self, subtype.coe_eta, subtype.coe_mk],
   λ _, by simp only [perm.apply_inv_self, subtype.coe_eta, subtype.coe_mk]⟩
 
+@[simp] lemma subtype_perm_apply (f : perm α) {p : α → Prop} (h : ∀ x, p x ↔ p (f x))
+  (x : {x // p x}) : subtype_perm f h x = ⟨f x, (h _).1 x.2⟩ := rfl
+
```

Comment:
Instead of adding this, you can just put `@[simps]` on the previous `def` and it will generate it for you (I think)

Reply:
`@[simps]` adds:
```
equiv.perm.subtype_perm_apply_coe:
        > ∀ {α : Type u} (f : perm α) {p : α → Prop} (h : ∀ (x : α), p x ↔ p (⇑f x)) (x : {x // p x}),
  ↑(⇑(f.subtype_perm h) x) = ⇑f ↑x
```
which is a bit different.. Should I do both?

Reply:
I think you can pass `{fully_applied := ff}` to do what you had originally, but don't worry about it if you can't find the right spelling, what you have in the PR is fine if the alternative is hard

Reply:
I think I'll leave it as is. At least it's explicit what the simp lemma is...